Mathlib/Algebra/Group/Submonoid/Membership.lean

```diff
@@ -482,40 +483,6 @@ an additive group if that element has finite order."] Submonoid.groupPowers
 
 end AddSubmonoid
 
-/-! Lemmas about additive closures of `Subsemigroup`. -/
```

Comment:
Please mention the changes to this file in the PR description.

Reply:
Sorry, those changes were added by accident. They belong in #20748